### ISSUE TITLE
`make html` warnings

### ISSUE THREAD

**Issue by joshmoore** [ome/omero-guides#56]
* /opt/own-guides/omero-guides/upload/docs/import-cli.rst:24

```
 .. _Resources:

Resources
---------
```

Should underscore anchors will need to be renamed _per guide_.

* /opt/own-guides/omero-guides/iviewer/docs/iviewer_rois.rst:133

```
.. |image16| image:: images/Iviewer16.png
   :width: 0.22917in
   :height: 0.1875in
```
16 doesn't exist:
```
(base) /opt/own-guides/omero-guides $find . -name Iviewer1*.png
./_build/html/_images/Iviewer1.png
./_build/html/_images/Iviewer11.png
./_build/html/_images/Iviewer10.png
./_build/html/_images/Iviewer12.png
./_build/html/_images/Iviewer13.png
./_build/html/_images/Iviewer14.png
./_build/html/_images/Iviewer1b.png
./_build/html/_images/Iviewer1c.png
./_build/html/_images/Iviewer15.png
./iviewer/docs/images/Iviewer1.png
./iviewer/docs/images/Iviewer11.png
./iviewer/docs/images/Iviewer10.png
./iviewer/docs/images/Iviewer12.png
./iviewer/docs/images/Iviewer13.png
./iviewer/docs/images/Iviewer14.png
./iviewer/docs/images/Iviewer1b.png
./iviewer/docs/images/Iviewer1c.png
./iviewer/docs/images/Iviewer15.png
```

* /opt/own-guides/omero-guides/example.rst:54:

```
-**Setup:**
-----------
+Setup
+-----
```
This text has been changed.

----

Migrated from https://github.com/ome/omero-guides/pull/54

**Comment by joshmoore** [comment]
@pwalczysko : :+1: for the PRs. Do you expect there to be no warnings at the moment? If so, shall we make warnings errors in the GHA builds?

**Comment by pwalczysko** [comment]
> Do you expect there to be no warnings at the moment? 

No. One of the warnings is because of the relative path to a file.

``upload/docs/import-cli.rst:47: WARNING: download file not readable: /Users/pwalczysko/Work/scripts/bulk.yml``

this warning is produced when the ``make html`` is run locally from the top level repo (``omero-guides``) but the file is fine on https://omero-guides.readthedocs.io/projects/upload/en/latest/import-cli.html?highlight=bulk.yml#bulk-import-using-the-cli -  you can download https://omero-guides.readthedocs.io/projects/upload/en/latest/_downloads/904615f9c13e025de80ada2f34b1d5ae/bulk.yml as expected.

The rest should nevertheless be clear - the only confusion I have is that I did not see some of the warns you reported.

**Comment by pwalczysko** [comment]
> shall we make warnings errors in the GHA builds

this would make sense but depending on your judgement Re relative filepaths as indicated in https://github.com/ome/omero-guides/issues/56#issuecomment-1177816297 - atm I am deeming it a "must live with that warning because no better idea and principally it works", but open to suggestions
